shared/chat/conversation/input-area/normal/container.js

```diff
@@ -102,7 +102,6 @@ const mergeProps = (stateProps, dispatchProps, ownProps: OwnProps): Props => ({
     } else {
       dispatchProps._onPostMessage(stateProps.conversationIDKey, text)
     }
-    ownProps.onScrollDown()
```

Comment:
scrlldown is based on if you send something last and not a side effect of sending